README.md

```diff
@@ -46,10 +47,37 @@ The file structure expected for your application or library.
 ```
 
 
+## Config file
+
+* Systematic requires a `systematic.ini` configuration file in the root folder of your project. Find out all the available options in `[systematic.example.ini](blob/master/systematic.example.ini)`
+
+
+## Makefile
+
+Create a Makefile at the root of your project, to import systematic commands:
+
+  ```makefile
+  include node_modules/systematic/mk/main.mk
+
+  # Optionally
+  include node_modules/systematic/mk/YOUR_FRAMEWORK.mk
```

Comment:
not sure :) today, systematic includes the build profile makefile itself, according to the one defined in the systematic.ini.